src/domain/useCase/createDatabase.go

```diff
@@ -13,7 +13,7 @@ func CreateDatabase(
 	dbCmdRepo repository.DatabaseCmdRepo,
```

Comment:
You edited a few usecases but they continue to use log.Print instead of slog. Please replace the log instances on the usecases edited.

Reply:
Done.